docs/source/how-to/deploying-substra/upgrade-notes.rst

```diff
@@ -4,6 +4,93 @@
 Upgrade notes
 *************
 
+.. _ops upgrade notes 0.34:
+
+Substra 0.34.0
+--------------
+
+This version upgrades the integrated PostgreSQL databases from versions 14 to 16. This upgrade must be performed manually.
+
+If you :ref:`do not use the integrated databases <ops howto external database>`, then you have nothing to do.
+
+Otherwise, you must update the data format. There are many ways to do this; here we give you a simple way which is acceptable if you have small amounts of data, you have access to the clusters, and downtime is acceptable. It must be repeated on each Substra node and orchestrator.
```

Comment:
I would emphase on the requirements somehow:
```suggestion
Otherwise, you must update the data format. There are many ways to do this; here we give you a simple way which is acceptable if:
  - Have small amounts of data
  - Have access to the clusters
  - Downtime is acceptable
 It must be repeated on each Substra node and orchestrator.
```